plugins/poll/plugin.rb

```diff
@@ -435,6 +437,16 @@ def voters
     end
   end
 
+  on(:reduce_excerpt) do |fragment|
+    fragment.css("div.poll").each(&:remove)
+  end
+
+  on(:reduce_onebox) do |onebox, post|
```

Comment:
I don't think we need to handle oneboxes differently. We should use the excerpt, and if there's a poll in the excerpt, just replace it with a link to the post.